nipoppy_cli/tests/test_workflow_dicom_reorg.py

```diff
@@ -451,11 +483,38 @@ def test_run_main(
         ).validate(),
     ],
 )
-def test_cleanup(doughnut: Doughnut, tmp_path: Path):
+def test_cleanup_doughnut(doughnut: Doughnut, tmp_path: Path):
     workflow = DicomReorgWorkflow(dpath_root=tmp_path / "my_dataset")
     workflow.doughnut = doughnut
 
     workflow.run_cleanup()
 
     assert workflow.layout.fpath_doughnut.exists()
     assert Doughnut.load(workflow.layout.fpath_doughnut).equals(doughnut)
+
+
+@pytest.mark.parametrize(
+    "n_success,n_total,expected_message",
+    [
+        (0, 0, "No participant-session pairs to reorganize"),
+        (0, 1, f"[{LogColor.FAILURE}]Reorganized files for"),
+        (1, 2, f"[{LogColor.PARTIAL_SUCCESS}]Reorganized files for"),
+        (2, 2, f"[{LogColor.SUCCESS}]Successfully reorganized files for"),
```

Comment:
I would include the full expected message to be more robust and substitute the `n_success` and `n_total` during the assert. e.g. `f"[{LogColor.FAILURE}]Reorganized files for {0} out of {1} participant-session pairs"` then assert for `expected_message.format(n_success, n_total)`.

In case we decide to change the color in the future, this could prevent the test to pass silently.